docs/livereload.md

```diff
@@ -1,6 +1,8 @@
 # Start livereloading and asset building on localhost:3000
 
-When you have docker-compose running in one terminal window please open a second window where you run `npm run dev`. This will start a process that will be building our frontend dependencies. Our build setup provides a [`localhost:3000`](http://localhost:3000) address that shows the same as localhost:5000 but also has livereloading.
+Update the `proxy`-address in [`webpack.config.js`](../webpack.config.js) to the same as `PRIMARY_SITE_URL` in `.env`.
```

Comment:
Kva med å hente denne verdien automatisk fra process.env.PRIMARY_SITE_URL i webpack.config.js ?

Reply:
Dette fungerer ikke direkte dessverre, det som åpner seg i nettleseren da er http://localhost:3001 og der er BrowserSync og hvis jeg går til http://localhost:3000 så får jeg se mappeutlistingen av prosjektet 😅

Jeg har prøvd å følge:
- https://stackoverflow.com/questions/46224986/how-to-pass-env-file-variables-to-webpack-config
- https://webpack.js.org/plugins/environment-plugin/

Alle disse ga meg samme resultat.